tests/integration_tests/modules/test_ntp_servers.py

```diff
@@ -46,13 +46,17 @@ def test_dist_config_file_is_empty(
         (This test is skipped on all currently supported Ubuntu releases, so
         may not actually be needed any longer.)
         """
-        if class_client.execute("test -e /etc/ntp.conf.dist").failed:
-            pytest.skip("/etc/ntp.conf.dist does not exist")
-        dist_file = class_client.read_from_file("/etc/ntp.conf.dist")
+        if class_client.execute(
+            "test -e /etc/ntpsec/ntp.conf.dpkg-dist"
```

Comment:
Shouldn't we operate over `/etc/ntpsec/ntp.conf.dist`?

```suggestion
            "test -e /etc/ntpsec/ntp.conf.dist"
```

`/etc/ntpsec/ntp.conf.dist` is the file cloud-init renames if `/etc/ntpsec/ntp.conf` is present and `/etc/ntpsec/ntp.conf.dpkg-dist` is the one `apt` / `dpkg` creates on installation / upgrade paths.